data.sql

```diff
@@ -32,3 +32,24 @@ VALUES ('Boarmon',  '2005/06/07', '7', 'true', '20.4');
 INSERT INTO animals (animal_name, date_of_birth, escape_attempts, neutered, weight_per_kg)
 VALUES ('Blossom',  '1998/10/13', '3', 'true', '17');
 
+
+/* Insert Owners Names and Ages Into Owners Table */
+INSERT INTO owners (full_name, age) VALUES ('Sam Smith', 34);
+INSERT INTO owners (full_name, age) VALUES ('Jennifer Orwell', 19), ('Bob', 45);
+INSERT INTO owners (full_name, age) VALUES ('Melody Pond', 77);
+INSERT INTO owners (full_name, age) VALUES ('Dean Winchester', 14);
+INSERT INTO owners (full_name, age) VALUES ('Jodie Whittaker', 38);
+
+/* Insert Species Names Into Species Table. */
+INSERT INTO species (name) VALUES ('Pokemon'), ('Digimon');
+
+/* Update animals Table. */
+UPDATE animals SET species_id = (SELECT id FROM species WHERE name = 'Digimon') WHERE name LIKE '%mon';
+UPDATE animals SET species_id = (SELECT id FROM species WHERE name = 'Pokemon') WHERE name NOT LIKE '%mon';
```

Comment:
[OPTIONAL]

- Great implementation! 👍
Here 👇 is another way you can `UPDATE` the `animals` table

```
 UPDATE animals
   SET species_id = (SELECT id from species WHERE name = 'Pokemon')
   WHERE species_id IS NULL;
```

Reply:
Thanks for the correction @DammyShittu